kubernetes/storage.yaml

```diff
@@ -70,11 +82,16 @@ metadata:
     prometheus.io/scrape: "true"
   name: storage
   namespace: buildbarn
+  labels:
```

Comment:
What's the purpose of this change and the one below? The same holds for the changes to frontend.yaml.